File: DSA/DSA06011 TỔNG GẦN 0 NHẤT.cpp
#include <bits/stdc++.h>
using namespace std;



void submit()
{
    int n;
    cin >> n;
    vector<int> A(n);
    for (auto &&i : A) cin >> i;

    long long mini = INT32_MAX;

    for (int i = 0; i < n-1; i++)
    {
        for (int j = i+1; j < n; j++)
        {
            if (abs(A[i] + A[j]) < abs(mini))
            {
                mini = A[i] + A[j];
            }
            
        }
        
    }
    
    cout << mini;

    cout << endl;
}


int main()
{
    ios::sync_with_stdio(false);                                    // Speed up input/output
    cin.tie(nullptr);
    cout.tie(nullptr);
    
    int so_bo_test;
    cin >> so_bo_test;
    cin.ignore();
    
    while (so_bo_test--)
    {
        submit();
    }

    return 0;
}

File: DSA_python/DSA06011 TỔNG GẦN 0 NHẤT.py
def submit():
    n = int(input())
    A = list(map(int, input().split()))

    mini = 1000000000
    for i in range(n-1):
        for j in range(i+1, n):
            if abs(A[i]+A[j]) < abs(mini):
                mini = A[i]+A[j]

    print(mini)



test = int(input())
for i in range(test):
    submit()
